### ISSUE TITLE
URL generation method bug

### ISSUE THREAD

**Issue by kashish26** [imagekit-developer/imagekit-python#12]
```
from imagekitio import ImageKit
imagekit = ImageKit(
    private_key='YOUR_PRIVATE_KEY',
    public_key='YOUR_PUBLIC_KEY',
    url_endpoint = 'https://ik.imagekit.io/imagekit_id/'
)

imagekit.url({
    "path": "/coffee.jpg",
    "signed": False,
    "expire_seconds": 10
})
```

The above code produces the below output. The unrequired 'tr:' **path** parameter leads to a **'Not Found'** error. Ideally, the tr parameter should be optional. Using **src** variable also leads to 'tr:' query parameter in the URL.

https://ik.imagekit.io/imagekit_id/tr:/coffee.jpg?ik-sdk-version=python-2.2.5

**Comment by ahnv** [comment]
Fixed and Merge to master

**Comment by ahnv** [comment]
Released in [Release 2.2.6](https://pypi.org/project/imagekitio/2.2.6/)
